{
    "Computer":  "VSDC1",
    "Events":  [
                   {
                       "TimeCreated":  "\/Date(1759542221438)\/",
                       "Source":  "Microsoft-Windows-TaskScheduler",
                       "LogName":  "Microsoft-Windows-TaskScheduler/Operational",
                       "Level":  "Error",
                       "Message":  "Task Scheduler failed to start instance \"{a860543f-91cd-4694-9557-d6266e9a1317}\" of \"\\Microsoft\\Windows\\.NET Framework\\.NET Framework NGEN v4.0.30319\"  task for user \"NT AUTHORITY\\SYSTEM\" . Additional Data: Error Value: 2147946720.",
                       "Id":  103,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  152360
                   },
                   {
                       "TimeCreated":  "\/Date(1759542221438)\/",
                       "Source":  "Microsoft-Windows-TaskScheduler",
                       "LogName":  "Microsoft-Windows-TaskScheduler/Operational",
                       "Level":  "Error",
                       "Message":  "Task Scheduler failed to complete task \"\\Microsoft\\Windows\\.NET Framework\\.NET Framework NGEN v4.0.30319\" , instance \"{a860543f-91cd-4694-9557-d6266e9a1317}\" , action \"\" . Additional Data: Error Value: 2147946720.",
                       "Id":  202,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  152359
                   },
                   {
                       "TimeCreated":  "\/Date(1759539830515)\/",
                       "Source":  "Microsoft-Windows-TaskScheduler",
                       "LogName":  "Microsoft-Windows-TaskScheduler/Operational",
                       "Level":  "Error",
                       "Message":  "Task Scheduler failed to start \"\\Microsoft\\Windows\\Workplace Join\\Automatic-Device-Join\" task for user \"(default user)\". Additional Data: Error Value: 2147943568.",
                       "Id":  101,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  152175
                   },
                   {
                       "TimeCreated":  "\/Date(1759539786277)\/",
                       "Source":  "Microsoft-Windows-Perflib",
                       "LogName":  "Application",
                       "Level":  "Error",
                       "Message":  "Access to performance data was denied to user \"SYSTEM\" (value from GetUserName() for the running thread) as attempted from module \"C:\\Program Files (x86)\\NinjaOne\\NinjaRMMAgent.exe\" (value from GetModuleFileName() for the binary that issued the query).",
                       "Id":  1000,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  79384
                   },
                   {
                       "TimeCreated":  "\/Date(1759539771058)\/",
                       "Source":  "Microsoft-Windows-DistributedCOM",
                       "LogName":  "System",
                       "Level":  "Error",
                       "Message":  "DCOM got error \"1115\" attempting to start the service TrustedInstaller with arguments \"Unavailable\" in order to run the server:\r",
                       "Id":  10005,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  192493
                   },
                   {
                       "TimeCreated":  "\/Date(1759539731985)\/",
                       "Source":  "Microsoft-Windows-TaskScheduler",
                       "LogName":  "Microsoft-Windows-TaskScheduler/Operational",
                       "Level":  "Error",
                       "Message":  "Task Scheduler failed to launch action \"powershell.exe\" in instance \"{fccea7c6-c98d-43ac-af11-cff7d035a9b6}\" of task \"\\ServerHealthMonitor\". Additional Data: Error Value: 2147943515.",
                       "Id":  203,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  152161
                   },
                   {
                       "TimeCreated":  "\/Date(1759539666859)\/",
                       "Source":  "Service Control Manager",
                       "LogName":  "System",
                       "Level":  "Error",
                       "Message":  "The Update Orchestrator Service service terminated with the following error: \r",
                       "Id":  7023,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  192483
                   },
                   {
                       "TimeCreated":  "\/Date(1759539557327)\/",
                       "Source":  "Microsoft-Windows-Perflib",
                       "LogName":  "Application",
                       "Level":  "Error",
                       "Message":  "Windows cannot load the extensible counter DLL \"C:\\Windows\\system32\\ntdsperf.dll\" (Win32 error code The specified module could not be found.).",
                       "Id":  1023,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  79379
                   },
                   {
                       "TimeCreated":  "\/Date(1759508237307)\/",
                       "Source":  "Microsoft-Windows-Kerberos-Key-Distribution-Center",
                       "LogName":  "System",
                       "Level":  "Error",
                       "Message":  "This event indicates an attempt was made to use smartcard logon, but the KDC is unable to use the PKINIT protocol because it is missing a suitable certificate.",
                       "Id":  19,
                       "ErrorCode":  null,
                       "UpdateTitle":  null,
                       "EventRecordID":  191835
                   },
                   {
                       "TimeCreated":  "\/Date(1759505001252)\/",
                       "Source":  "VSS",
                       "LogName":  "Application",
                       "Level":  "Error",
                       "Message":  "Volume Shadow Copy Service error: Error calling a routine on a Shadow Copy Provider {74600e39-7dc5-4567-a03b-f091d6c7b092}. Routine details BeginPrepareSnapshot({3a35e323-5c3d-4660-9ac0-eec83c446560},{035b5483-fca9-4442-92c9-8c1559d57d2a},\\\\?\\Volume{c6def331-2795-4a00-bf35-64cc5b01fbec}\\) [hr = 0x8000ffff, Catastrophic failure\r",
                       "Id":  12293,
                       "ErrorCode":  "0x8000ffff",
                       "UpdateTitle":  null,
                       "EventRecordID":  78925
                   }
               ],
    "LastPoll":  "10/03 19:47"
}